daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/Util.scala

```diff
@@ -84,4 +84,69 @@ object Util {
   val CPTrue = CPPrimCon(PCTrue)
   val CPFalse = CPPrimCon(PCFalse)
 
+  type Graph[X] = Map[X, Set[X]]
```

Comment:
Petition to move these to their own utility class that isn't called "Util" or something. How about `Graphs`?

You could even wrap it in a `case class Graph[X](graph: Map[X, Set[X]]) extends AnyVal` and make them methods. I would find that much more readable.

Reply:
Fair enough, good point.